src/util/std_expr.h

```diff
@@ -2539,7 +2538,8 @@ class shift_exprt:public binary_exprt
 template <>
 inline bool can_cast_expr<shift_exprt>(const exprt &base)
 {
-  return base.id() == ID_shl || base.id() == ID_ashr || base.id() == ID_lshr;
+  return base.id() == ID_shl || base.id() == ID_ashr || base.id() == ID_lshr ||
+         base.id() == ID_ror || base.id() == ID_rol;
```

Comment:
There are `lshr_exprt` et al. so it would be good to have `ror_exprt` and `rol_exprt` for consistency.